srcpkgs/bucklespring/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'bucklespring'
+pkgname=bucklespring
+version=1.4.0
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+make_build_args="PATH_AUDIO=/usr/share/${pkgname}/wav"
+hostmakedepends="pkg-config"
+makedepends="alure-devel libinput-devel libXtst-devel"
+depends="bucklespring-data"
+short_desc="Emulate the sound of the IBM Model-M while typing"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
+license="GPL-2.0-only"
+homepage="https://github.com/zevv/bucklespring"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=b38b2c0ec905573a243cb9db543f8eeb92e4d040025ef338d8bcfcfd5ccd5348
+
+if [ $XBPS_CROSS_BASE ]; then
+	make_build_args+=" CROSS=$XBPS_CROSS_TRIPLET-"
```

Comment:
isn't there an extra dash at the end or is it required ?

Reply:
Yeah, they do this stupid thing: 

~~~
CC 	?= $(CROSS)gcc
~~~

For example in their own makefile:

~~~
 CROSS   := i686-w64-mingw32-
~~~

Reply:
I guess I could just just do CC=$CC

Reply:
I thought $CC or $CC_FOR_BUILD would give me the right cross compiler?